src/middleware.js

```diff
@@ -1,18 +1,41 @@
 const facts = require("./models/facts");
-const { convert } = require("sst");
 
-const ISO_LANG = ["eng-us", "ukr-ua", "rus-ru", "esp-mx", "esp-es"];
-const SHORT_LANG = ["eng", "ukr", "rus", "esp"];
+// We include all supported language / location combinations here:
+const ISO_LANG = [
+  "eng-us",
+  "ukr-ua",
+  "rus-ru",
+  "esp-mx",
+  "esp-es",
+  "ces-cz",
+  "cze-cz",
+];
+
+// we also allow all included short ISO 639-2 codes used without location
+// to support backwards compatibility
+const SHORT_LANG = ["eng", "ukr", "rus", "esp", "ces", "cze"];
 const VALID_LANGUAGES = SHORT_LANG.concat(ISO_LANG);
 
+/**
+ * parse string to number. returns NaN if string can not be parsed to number
+ * @param {Any} num
+ * @param {Number} precision
+ * @returns {Number} 
+ */
+function toNumber(num, precision) {
```

Comment:
this is a breakout of a few utils functions. noted in https://github.com/wh-iterabb-it/meowfacts/issues/195